### PR TITLE
Early review of part 1 & part 2

### PR DESCRIPTION
**Pull request by SueChaplain**
I see this guide in 2 parts:

1. How to create, update, publish, release an application stack.
2. How to configure your CRD

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Comment by SueChaplain** [comment]
@neeraj-laad  .... please can you take a look from the Appsody perspective.

**Comment by SueChaplain** [comment]
@kaczyns - Please can you provide some input for the CRD part of this guide.

**Comment by stephenkinder** [comment]
Just FYI for those looking:  https://github.com/SueChaplain/draft-guide-working-with-stacks/tree/review

**Comment by SueChaplain** [comment]
>Do we use Queen's English or American English? organisation

@stephenkinder  - Old habits die hard! This would make my English school teacher very sad ;-)
Fixed!

**Comment by SueChaplain** [comment]
@stephenkinder @kaczyns 

I've now added my best guess for configuring the CRD. I used the information in the design and the latest `full.yaml` sample .... there are some slight differences between the two, so please let me know what needs changing.

The ending could be improved too. I've not had any hands-on experience of the operator .... please feel free to dictate some words.

**Comment by SueChaplain** [comment]
Thanks for your review @kaczyns - all your changes so far are applied. Let me know if anything else is needed.

File: README.md [hunk 1]
```diff
@@ -32,8 +33,8 @@ guide-category: collections
 
 ## What you'll learn
 
-In this guide you will learn how to create a local stack hub for the application stacks that you want to use in your organisation. Then you'll learn how to create a new stack, customize it, and publish it to a
-GitHub repository for developers to use in their local development environment. Finally you'll learn how to configure your kubernetes custom resource definition (CRD) to initialize and deploy the containerized applications that are developed.  
+In this guide you will learn how to create a local stack hub for the application stacks that you want to use in your organization. Then you'll learn how to create a new stack, customize it, and publish it to a
```
Comment (davehobbs) [inline README.md]: Although stack hub is mentioned here and in the getting started section there is no subsequent mention of stack hubs later in the guide. Is that a subject for another guide?

Comment (SueChaplain) [reply README.md]: Dave and I discussed this and agreed that the mention of "stack hub" in this guide, specifically as the "control point" for application stacks was not useful. I have modified the "What you'll learn" and "Getting Started" sections accordingly. 

Please can @stephenkinder and @kilnerm or @groeges take a quick look and confirm that the updates are ok?

**Pull request merged**